runhouse/servers/cluster_servlet.py

```diff
@@ -419,22 +450,34 @@ async def astatus(self):
                 env_servlet_utilization_data[env_servlet_name] = env_memory_info
 
         # TODO: decide if we need this info at all: cpu_usage, memory_usage, disk_usage
-        cpu_usage = psutil.cpu_percent(interval=1)
+        cpu_utilization = psutil.cpu_percent(interval=1)
 
         # Fields: `available`, `percent`, `used`, `free`, `active`, `inactive`, `buffers`, `cached`, `shared`, `slab`
         memory_usage = psutil.virtual_memory()._asdict()
 
         # Fields: `total`, `used`, `free`, `percent`
         disk_usage = psutil.disk_usage("/")._asdict()
 
+        server_pid: int = get_pid()
+
+        # get general gpu usage
+        server_gpu_usage = (
+            self._get_node_gpu_usage(server_pid)
+            if self.cluster_config.get("has_cuda", False)
+            else {}
```

Comment:
Small nit, would it be better to return `None` here? Makes the logic a bit easier on the front-end (can check if missing instead of if missing OR missing values)